web/src/containers/admin/MyAccount.js

```diff
@@ -75,66 +82,99 @@ class MyAccount extends Component {
-                    ariaLabel="please enter your full name"
-                    value={name || ''}
-                    onChange={this.handleEditAccount}
-                  />
-                  <TextField
-                    label="Phone number"
-                    ariaLabel="please enter your 10-digit phone number"
-                    mask="phone"
-                    name="phone"
-                    size="medium"
-                    value={phone || ''}
-                    onChange={this.handleEditAccount}
-                  />
-                  <TextField
-                    label="Role"
-                    name="position"
-                    ariaLabel="please enter your position or role ???"
-                    value={position || ''}
-                    onChange={this.handleEditAccount}
-                  />
-                  <Password
-                    showMeter
-                    className="mb2"
-                    title="Change password"
-                    value={password}
-                    onChange={this.handleEditAccount}
-                  />
-                  <Button variation="primary" type="submit" disabled={fetching}>
-                    {fetching ? (
-                      <Fragment>
-                        <Spinner /> Working
-                      </Fragment>
-                    ) : (
-                      'Save changes'
+          <div className="ds-l-container">
+            <div className="ds-l-row card">
+              <div className="ds-l-col--1 ds-u-margin-left--auto" />
+              <div className="ds-l-col--12 ds-l-sm-col--10 ds-l-lg-col--6">
+                {!!success && <Alert variation="success">Changes saved</Alert>}
+
+                <h1 className="ds-h1">Manage account</h1>
+
+                <form onSubmit={this.editAccount}>
+                  <fieldset className="ds-u-margin--0 ds-u-padding--0 ds-u-border--0">
+                    <legend className="sr-only">manage account details</legend>
+                    <TextField
+                      label="Name"
+                      name="name"
+                      ariaLabel="please enter your full name"
+                      value={name || ''}
+                      onChange={this.handleEditAccount}
+                    />
+                    <TextField
+                      label="Phone number"
+                      ariaLabel="please enter your 10-digit phone number"
+                      mask="phone"
+                      name="phone"
+                      size="medium"
+                      value={phone || ''}
+                      onChange={this.handleEditAccount}
+                    />
+                    <TextField
+                      label="Role"
+                      name="position"
+                      ariaLabel="please enter your position or role ???"
+                      value={position || ''}
+                      onChange={this.handleEditAccount}
+                    />
```

Comment:
This comment is for the line starting below.  Our conversation on #1397 is probably relevant to this one as well.  I put the disabled password field and the "change password" button on a row so that they'd line up.

On the button side, I added a `div` with the label classes to provide the right vertical spacing and wrapped the button in a `div` styled as a field to get its spacing right, too.  Hopefully that's not asinine.